keyboards/handwired/dactyl_rah/keymaps/right/keymap.c

```diff
@@ -0,0 +1 @@
+../default/keymap.c
```

Comment:
Not sure what this is...

Reply:
I have both USB's exposed on the left and right side of the keyboard so I can flash from either side. Instead of having two separate files, I created a symlink so that I only need to edit one file for the keymap for both left and right. Github review seem to show it as `../default/keymap.c`

Reply:
We generally don't allow symlinks since they don't work quite right on Windows. Please just make a copy.

Reply:
Done